internal/idp/keycloak/types.go

```
@@ -206,3 +206,83 @@ func fromKeycloakOrganization(kcOrg *keycloakOrganization) *idp.Organization {
+
+	return &keycloakAuthorizationResource{
+		ID:         resource.ID,
+		Name:       resource.Name,
+		Type:       resource.Type,
+		Scopes:     scopes,
+		URIs:       resource.URIs,
+		Attributes: resource.Attributes,
+	}
+}
+
+func fromKeycloakAuthorizationResource(kcResource *keycloakAuthorizationResource) *AuthorizationResource {
```

golangci-lint (2.12.2)

[error] func fromKeycloakAuthorizationResource is unused

(unused)